### ISSUE TITLE
Merge snippet folders to avoid filename conflicts

### ISSUE THREAD

**Issue by nether-cat** [PAYONE-GmbH/shopware-5#54]
There are two snippet folders in the same directory path:

`Frontend/MoptPaymentPayone/Snippets`
`Frontend/MoptPaymentPayone/snippets`

That causes filename conflicts on case-insensitive filesystems (e.g. on Windows and macOS). I am suggesting to merge the two directories to avoid these conflicts.

**Comment by fjbender** [comment]
Good idea. Go ahead.

Can we address the deprecation of the "Default" snippet group when doing this? It's been on the list in SW-49 for quite some time now:
```
1) Shopware_Tests_Components_Snippet_SnippetStructureTest::testSnippetsShouldBeValid
Snippet validation errors detected: 
"Default" snippet group is deprecated (/home/florian/Dev/shopware/engine/Shopware/Plugins/Default/Frontend/MoptPaymentPayone/Snippets/frontend/MoptPaymentPayone/payment.ini)
[...]
```

This would enable us to run Unit Tests automatically through Travis, as this is the last issue blocking this.

**Comment by fjbender** [comment]
Fixed in #55.
